c/pthread-lit/fkp2013_false-unreach-call.i

```diff
@@ -654,12 +654,9 @@ void __VERIFIER_assert(int cond) {
   }
   return;
 }
-void __VERIFIER_atomic_assert(int cond) {
```

Comment:
I think it was used in one or two files, however, see my other comment regarding `__VERIFIER_atomic_assert`...  (Basically, I don't think that the atomic version is logically any different than a non-atomic version).